aiohttp/worker.py

```diff
@@ -159,6 +161,11 @@ def _create_ssl_context(cfg):
             ctx.set_ciphers(cfg.ciphers)
         return ctx
 
+    def _fix_log_format(self, source_format, default_format):
+        if re.search(r'%\([^\)]+\)', source_format):
+            return default_format
```

Comment:
Oh, no.
Silent format replacement should be performed only if source format is exactly *default gunicorn log format*, `%(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"` according to the doc.
For all other gunicorn-styled but not default formats `ValueError` should be raised.